app/src/main/java/de/hbch/traewelling/ui/dashboard/DashboardFragment.kt

```diff
@@ -60,7 +56,7 @@ class DashboardFragment : Fragment() {
         searchStationCard.requestPermissionCallback = { permission ->
             requestPermissionLauncher.launch(permission)
         }
-        loggedInUserViewModel.getLoggedInUser()
+        loggedInUserViewModel.login()
```

Comment:
I think in this context, the method name login() could be misleading because there happens no log in. Just the current logged in user will be requested by this function.